test/e2e/etcd_backup_test.go

```diff
@@ -70,7 +70,8 @@ var _ = Describe("Etcd Backup", func() {
 					Expect(err).ShouldNot(HaveOccurred())
 
 					// purge any existing backups in bucket
-					Expect(purgeSnapstore(store)).To(Succeed())
+					// Expect(purgeSnapstore(store)).To(Succeed())
+					purgeSnapstore(store)
```

Comment:
I think these two lines should be reverted

Reply:
Corrected